ExpressionEngine/FunctionDefinition.cs

```diff
@@ -0,0 +1,41 @@
+namespace ExpressionEngine
+{
+    /// <summary>
+    /// A Function Definition is a map to a collected set of functions.
+    /// - currentCellValue() -> formvalue(logicalName())[idx()][attributeLogicalName()]
+    /// </summary>
+    public class FunctionDefinition : IFunctionDefinition
+    {
+        private readonly string _from;
+        private readonly string _to;
+
+        /// <summary>
+        /// Construction a Function Definition
+        /// </summary>
+        /// <param name="from"></param>
+        /// <param name="to"></param>
+        public FunctionDefinition(string from, string to)
+        {
+            _from = @from;
+            _to = to;
+        }
+
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <returns></returns>
+        public string From()
```

Comment:
I noticed this somewhere else also in expressionengine, that you use methods to something that I think the general c#/dotnet developer would use properties.  I actually think this is what you would expect see more in Java.  

public string From {get;set;} 

@sjkp - is this just me? or would you also expect a property more than a method here?

Reply:
I agree :)